contrib/python/test/test_containers.py

```diff
@@ -65,8 +63,22 @@ def test_get(self):
             self.pclient.containers.get("bozo")
 
     def test_attach(self):
-        with self.assertRaisesNotImplemented():
-            self.alpine_ctnr.attach()
+        # StringIO does not support fileno() so we had to go old school
+        input = os.path.join(self.tmpdir, 'test_attach.stdin')
+        output = os.path.join(self.tmpdir, 'test_attach.stdout')
+
+        with open(input, 'w+') as mock_in, open(output, 'w+') as mock_out:
+            # double quote is indeed in the expected place
+            mock_in.write('echo H"ello, World"; exit\n')
```

Comment:
Minor nit, but you probably meant `"H` instead of `H"`.

Reply:
I think based on his comment, he has it where he wants it?

Reply:
> I think based on his comment, he has it where he wants it?

Ah, I somehow missed the comment.  Out of curiosity, why put it there?

Reply:
@wking I wanted the target in the return stream to be the words 'Hello, World' from the echo command not the characters being printed from the shell 'H"ello, World'